contractsV5/FeeHandler.sol

```diff
@@ -220,16 +234,21 @@ contract FeeHandler is IFeeHandler, Utils {
 
     // if no one voted for an epoch (like epoch 0). no one gets reward. so should burn it.
     function shouldBurnEpochReward(uint epoch) public {
-        if (!kyberDAO.shouldBurnRewardForEpoch(epoch)) return;
+        require(address(kyberDAO) != address(0), "kyberDAO addr missing");
```

Comment:
Better to revert so sender could read the error message (instead of seeing tx is success but actually nothing happens)